automatic_cross_compile.sh

```diff
@@ -0,0 +1,61 @@
+# This script is intended to be used by the Jenkins night builds.
```

Comment:
missing `#/bin/bash` ?

Reply:
I would also prefer if all shell scripts that are executed by the user contain a shebang and the executable flag.